### PR TITLE
Chinese translation  modify 

### PR DESCRIPTION
**Pull request by zhaoying818**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/master/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----
This PR is to modify some translation of best-practices to Chinese.
- Place one space before/after English words 
- Place one space before/after numbers
- modify some translation  in Chinese

### CODE REVIEW

**Comment by mlinksva** [comment]
Thanks for this PR! I suspect that if these spacing around numbers and English words are best practices, they should be implemented for every article. @zhaoying818 can you point to any resources documenting that these are translation best practices?

**Comment by zhaoying818** [comment]
> can you point to any resources documenting that these are translation best practices?

I follow the following article
https://github.com/sparanoid/chinese-copywriting-guidelines/blob/master/README.en-US.md

**Comment by mlinksva** [comment]
Thanks for the reference!

Would another Chinese writer be willing to review this PR? Maybe a previous contributor to the article this PR changes -- @lijiangsheng1 @WillemJiang @lc-soft ?

File: _articles/zh-hans/best-practices.md [hunk 1]
```diff
@@ -231,29 +231,29 @@ fork一个项目不什么坏事情。能复制并且修改别人的代码是开
 * [mention-bot](https://github.com/facebook/mention-bot) 可能的贡献者来帮你复查代码
 * [Danger](https://github.com/danger/danger) 帮你自动复查代码
 
-对于bug报告和其他常见形式的贡献，Github有[Issue 模版和 Pull Request 模版](https://github.com/blog/2111-issue-and-pull-request-templates), 你可以用来降低沟通成本。你也可以设置[邮件过滤](https://github.com/blog/2203-email-updates-about-your-own-activity)来管理你的邮件提醒。
+对于 bug 报告和其他常见形式的贡献，Github 有[Issue 模版和 Pull Request 模版](https://github.com/blog/2111-issue-and-pull-request-templates), 你可以用来降低沟通成本。你也可以设置[邮件过滤](https://github.com/blog/2203-email-updates-about-your-own-activity)来管理你的邮件提醒。
 
-如果你想更加的先进和高效，代码风格指南和linter能让你项目收到的贡献更加规范，而且更容易复查和被接受。
+如果你想更加的先进和高效，代码风格指南和 linte r能让你项目收到的贡献更加规范，而且更容易复查和被接受。
```
Comment (lc-soft) [inline _articles/zh-hans/best-practices.md]:
```diff
- linte r
+ linter
```

File: _articles/zh-hans/best-practices.md [hunk 2]
```diff
@@ -231,29 +231,29 @@ fork一个项目不什么坏事情。能复制并且修改别人的代码是开
 * [mention-bot](https://github.com/facebook/mention-bot) 可能的贡献者来帮你复查代码
 * [Danger](https://github.com/danger/danger) 帮你自动复查代码
 
-对于bug报告和其他常见形式的贡献，Github有[Issue 模版和 Pull Request 模版](https://github.com/blog/2111-issue-and-pull-request-templates), 你可以用来降低沟通成本。你也可以设置[邮件过滤](https://github.com/blog/2203-email-updates-about-your-own-activity)来管理你的邮件提醒。
+对于 bug 报告和其他常见形式的贡献，Github 有[Issue 模版和 Pull Request 模版](https://github.com/blog/2111-issue-and-pull-request-templates), 你可以用来降低沟通成本。你也可以设置[邮件过滤](https://github.com/blog/2203-email-updates-about-your-own-activity)来管理你的邮件提醒。
```
Comment (lc-soft) [inline _articles/zh-hans/best-practices.md]:
```diff
- Github
+ GitHub
```

File: _articles/zh-hans/best-practices.md (hunk 1 above)
Comment (zhaoying818) [reply _articles/zh-hans/best-practices.md]: > ```diff
> - linte r
> + linter
> ```

modified，thx！

File: _articles/zh-hans/best-practices.md (hunk 2 above)
Comment (zhaoying818) [reply _articles/zh-hans/best-practices.md]: > ```diff
> - Github
> + GitHub
> ```

modified，thx！

**Pull request merged**